Plugins/IMA/IMAPlugin.swift

```diff
@@ -248,6 +233,7 @@ extension PKErrorDomain {
     private func updateAdTag(_ adTag: String, tagTimeKeyForRemove: TimeInterval) {
         self.tagsTimes![tagTimeKeyForRemove] = nil
         self.destroyManager()
+        self.contentComplete()
```

Comment:
This could be, let's flag this for refactor for your suggestion for separate handlers for manual and vmap.